HunabKu_urlshortener/hunabku_urlshortener/endpoints/Shortener.py

```diff
@@ -1,10 +1,19 @@
 from hunabku.HunabkuBase import HunabkuPluginBase, endpoint
 from hunabku.Config import Config, Param
 from flask import redirect
-from pymongo import MongoClient
+from pymongo import MongoClient, errors
 import validators
-from bson.objectid import ObjectId
+import datetime
+import base62
 
+<<<<<<< HEAD
+class Shortener(HunabkuPluginBase):
+    
+    counter = 0
+    last_sec = None
+=======
```

Comment:
same here

Reply:
My apologies for the error, I hope everything is ok now.